components/Loader.js

```diff
@@ -0,0 +1,40 @@
+import { View, Text, StyleSheet, useWindowDimensions, Dimensions, ActivityIndicator } from 'react-native'
+import React from 'react'
+import COLORS from '../const/colors'
+import SIZES from '../const/fontsize'
+
+
+const Loader = ({ visible = false }) => {
+    const { height, width } = Dimensions.get('screen');
+    return (
+        visible && (
+            <View style={[styles.container, { height, width }]}>

```

Comment:
You should use `Dimensions` directly in styles of `<View />` instead of destructuring them. Because height and width are only used once, there is no need to declare an additional line.